modules/apps/headless/headless-admin-address/headless-admin-address-test/src/testIntegration/java/com/liferay/headless/admin/address/resource/v1_0/test/CountryResourceTest.java

```diff
@@ -76,6 +83,55 @@ public void testGetCountriesPage() throws Exception {
 		assertValid(page);
 	}
 
+	@Override
+	@Test
+	public void testGetCountry() throws Exception {
+		super.testGetCountry();
+
+		Country country = testGetCountriesPage_addCountry(randomCountry());
+
+		Set<Locale> originalAvailableLocales =
+			_language.getCompanyAvailableLocales(
+				TestPropsValues.getCompanyId());
+
+		StringBuilder sb = new StringBuilder();
+
+		for (Locale locale : originalAvailableLocales) {
+			sb.append(
+				locale.toString()
+			).append(
+				","
+			);
+		}
```

Comment:
instead of this is not possible to save the originalUnicodeProperties? If it doesn't exists why at the end of the test we are resetting it instead of delete the "custom" property?

Reply:
I couldn't find a way to get the original Unicode Properties. And if we delete the custom property at the end it still doesn't work, the locales stay as "en_US,pt_BR"